tests/integration/test_row_fix.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+from src.trousse.dataframe_with_info import DataFrameWithInfo
+from trousse.row_fix import RowFix
+
+
+class Describe_RowFix:
```

Comment:
```suggestion
class DescribeRowFix:
```
No underscores in class names :)